lib/ret_web/channels/session_socket.ex

```diff
@@ -12,10 +12,25 @@ defmodule RetWeb.SessionSocket do
     "session:#{socket.assigns.session_id}"
   end
 
-  def connect(%{"session_id" => session_id}, socket) do
+  def connect(%{"session_token" => session_token}, socket) do
+    session_id =
```

Comment:
This seems worth a comment

Reply:
Hmm, not sure what you're getting at. The session token allows us to trust the session id contained in the token. Should I add a comment to the code saying that?

Reply:
Perhaps the moduledoc I just added is sufficient?